### PR TITLE
WIP: Structure Readings

### PR DESCRIPTION
**Pull request by brymz**
As per #209

### CODE REVIEW

File: readings/R-2.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+layout: page
+title: R Assignment 2
+subtitle: Reading
+language: R
+---
+
+__Topics__
```
Comment (ethanwhite) [inline readings/R-2.md]: This should be header.

File: readings/R-2.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+* Matrices & data frames
+* Importing data
+* Basic subsetting
+* Readable code
+
+__Reading__
```
Comment (ethanwhite) [inline readings/R-2.md]: This should be a header.

**Comment by ethanwhite** [comment]
I like the approach. Let me know when it's ready for a full review.

**Comment by brymz** [comment]
I've got through generating the readings files, updating the assignment files, and getting the readings files on the assignments/index.md. I can see assignments/index working as a table, but I don't know how to number rows. We could also add an image/button for `[Assignment]` and `[Reading]` links.

With the integration of readings and assignments, I'm not sure we also need a schedule page. Is there content on the schedule page you want to maintain? 

If you're happy with the restructuring, it should be ready to go. I'm not exactly sure where the merge conflict it (maybe git-1-R assignment title). Happy to take suggestions for the best approach to resolve.

**Pull request closed without merging**